pkg/deployments/tasks/20220908-weighted-pool-v2/readme.md

```diff
@@ -0,0 +1,13 @@
+# 2022-09-08 - Weighted Pool Factory V2
+
+Deployment of the `WeightedPoolFactory`, which allows creating Weighted Pools which include a number of improvements over [20210418-weighted-pool](../20210418-weighted-pool), including
```

Comment:
The link is broken since the other task is now in the `deprecated` dir. I'd however not bother linking to it and just mention the new stuff